biz.aQute.bndlib.tests/src/test/component/DSAnnotationTest.java

```diff
@@ -2,16 +2,19 @@
 
 import java.io.*;
 import java.util.*;
+import java.util.jar.*;
 import java.util.regex.*;
 
 import javax.xml.xpath.*;
 
 import org.osgi.framework.*;
+import org.osgi.framework.Constants;
```

Comment:
Delete this line. You should be using  aQute.bnd.osgi.Constants which is already imported below.